enhancements/console/quick-starts-crd.yaml

```diff
@@ -0,0 +1,144 @@
+    JSONPath: .spec.displayName
+  - name: Version
+    type: string
+    JSONPath: .spec.version
+  - name: Description
+    type: string
+    JSONPath: .spec.description
+  - name: Age
+    type: date
+    JSONPath: .metadata.creationTimestamp
+  subresources:
+    status: {}
+  "validation":
+    "openAPIV3Schema":
+      description: QuickStart is an extension for guiding user thought various workflows in the OpenShift web console
+        links.
+      type: object
+      required:
+      - spec
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: QuickStartSpec is the desired quick start configuration.
+          type: object
+          required:
+          - version
+          - displayName
+          - introduction
+          - tasks
+          properties:
+            version:
+              description: version definest the release version of the quick
+                start in semver format. Each Quick Start needs to be versioned,
+                to accuratelly reflect the workflow in particular release of
+                OpenShift's web console.
+              type: string
+            displayName:
+              description: displayName is the display name of the Quick Start.
+              type: string
+            iconURL:
+              description: iconURL is an URL of the Package Manifest that contains Quick Start's icon. (optional)
+              type: string
+            duration:
+              description: duration informs user on aproximately how long the
+                Quick Start should take to finish. (optional)
+              type: integer
+            description:
+              description: description is the description of the Quick Start. (optional, includes markdown)
+              type: string
+            prerequisites:
+              description: prerequisites contains a list prerequisites for the Quick Start. (optional)
+              type: array
+              items:
+                type: string
+            introduction:
+              description: introduction describes the purpose and usability of
+                the Quick Start. (includes markdown)
+              type: string
+            tasks:
```

Comment:
I see the tour content library getting really big, really quickly. That means effort duplication, staleness + accuracy problems, translation pains, etc. 

Could this definition support external content sources (say, [OCP docs modules](https://github.com/openshift/openshift-docs/tree/master/modules) so that tour tasks could be single sourced?

Reply:
> Could this definition support external content sources ...

@maxwelldb not sure what you mean by this. Do you suggest to import the fields from different CRDs ?